front-end/studio/src/app/pages/apis/{apiId}/editor/_components/common/inline-editor.base.ts

```diff
@@ -168,7 +175,6 @@ export abstract class TextInputEditorComponent extends AbstractInlineValueEditor
         this.input.changes.subscribe(changes => {
             if (changes.last) {
                 changes.last.nativeElement.focus();
-                changes.last.nativeElement.select();
```

Comment:
Is this important? When starting with an empty text field, it caused the first character typed to be immediately selected, and overwritten with the next one.

Reply:
I think auto selecting the value when there is a value is a nice feature.  Instead of removing that line can you check for a value and skip the line of the value is empty?

Reply:
Done!

Reply:
+1